Rest api req res format

weakish opened the pull request.

weakish committed:
feat(rest): synopsis for other services

- push
- installations
- schema
- cloud function
- rtm
- other

weakish committed:
feat(rest): request format

weakish committed:
feat(rest): response format

leeyeh commented:
Deploy preview for *lc-docs-en* ready!

Built with commit 47f5462937c3443a2a8df27cb8f9dd702384da0d

https://deploy-preview-137--lc-docs-en.netlify.com

weakish committed:
refactor(rest): remove unnecessary postman clips

Screenshots contain Chinese instructions,
and it is better to refer English readers to postman's official doc.
